ql/src/test/results/clientpositive/llap/dynamic_semijoin_reduction_3.q.out

```diff
@@ -814,30 +791,20 @@ STAGE PLANS:
                   Statistics: Num rows: 1 Data size: 8 Basic stats: COMPLETE Column stats: NONE
                   Group By Operator
                     aggregations: compute_stats(a, 'hll'), compute_stats(b, 'hll')
-                    minReductionHashAggr: 0.99
-                    mode: hash
+                    mode: complete
```

Comment:
It seems this complete mode is only happening for write type INSERT... I am wondering whether we are setting up the number of reducers to 1 too aggressively since INSERT does not need a single instance.

Reply:
Enabling parallelism when inserting eliminates this change.